gnews/gnews.py

```diff
@@ -200,7 +200,7 @@ def _clean(html):
 
     def _process(self, item):
         url = process_url(item, self._exclude_websites)
-        if url:
+        if url and (inspect.stack()[2][3] == 'get_news' or inspect.stack()[2][3] == 'get_news_by_site' or self._is_date_in_range(item.get("published", ""))):
```

Comment:
Hey @v-cardona can you please add test cases for this chnage?  also can you please update readme.md